src/public/modules/forms/admin/controllers/edit-fields-modal.client.controller.js

```diff
@@ -13,7 +13,8 @@ const { uploadImage } = require('../../../../services/FileHandlerService')
 const {
   DateSelectedValidation: DateValidationOptions,
 } = require('../../../../../shared/constants')
-
+const { Rating } = require('../../../../../types/form')
+const { RatingShape } = require('../../../../../types/field/ratingField')
```

Comment:
you can actually just import from `types`, since `types` contains an `index.ts` which exports everything